src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form class="form search-box-big"
+      action="#" 
+      method="post"
+      data-qa="big"
+    >
+      <input
+        class="input input-big" 
+        type="text"
+        name="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form class="form search-box-small"
+      action="#" 
+      method="post"
+      data-qa="small"
+    >
+      <input
+        class="input input-small" 
+        type="text"
+        name="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
```

Comment:
```suggestion
    
    <form 
      class="form"
      action="#" 
      method="post"
      data-qa="big"
    >
      <input
        class="input" 
        type="text"
        name="search"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
      >
    </form>

    <form 
      class="form form--big"
      action="#" 
      method="post"
      data-qa="small"
    >
      <input
        class="input input-small" 
        type="text"
        name="search"
        placeholder="Try &quot;Los Angeles&quot;"
      >
    </form>
```
lets make it work like this